charts/xrd-common/templates/_config-configmap.tpl

```diff
@@ -18,7 +18,7 @@ data:
     username {{ .Values.config.username }}
      group root-lr
      group cisco-support
-     password {{ .Values.config.password }}
+     secret {{ include "xrd.hasSecret" . }}
```

Comment:
```
RP/0/RP0/CPU0:ios(config-un)#secret ?
  0     Specifies a cleartext password will follow
  10    Specifies that SHA512-based password will follow
  8     Specifies that SHA256-based password will follow
  9     Specifies that Scrypt-based password will follow
  LINE  The cleartext user password
```

Given that we are using the LINE version of the secret configuration - i.e. the user enters a cleartext password - is there any need to deprecated .Values.config.password?  We can just change this to configure secret rather than password.

Reply:
My feeling was that it could be confusing to have the word "password" in Helm when the actual config is "secret", given that password also exists as config.

Reply:
On reflection, I don't think the complication of allowing both is worth it, have changed PR to just use secret.